.travis.yml

```diff
@@ -112,10 +113,10 @@ matrix:
       env: BUILD_ARGS="-DENABLE_PDF_DOCS=On -DENABLE_IPV6=On -DENABLE_LUA=On -DCMAKE_BUILD_TYPE=Debug"
     - os: osx
       compiler: gcc
-      env: BUILD_ARGS="-DENABLE_PDF_DOCS=On -DENABLE_IPV6=On -DENABLE_LUA=On -DGTK_BUILD_TYPE=GTK3"
+      env: BUILD_ARGS="-DENABLE_PDF_DOCS=On -DENABLE_IPV6=On -DENABLE_LUA=On -DGTK_BUILD_TYPE=GTK2"
```

Comment:
Why do you build on Travis using the GTK2 code which we don't fully maintain any further?
The GTK3 code is the most recent and maintained, so we should build using this code and GTK3 libraries.

Reply:
Meh, this is something leftover in the original test matrix.
In the previous version, we were defaulting to gtk2 builds, so the test matrix had to explicitly set gtk3 to test it. Now that we default to gtk3, that line became a complete no-op build. (already covered elsewhere).
So I see two solutions:
1) delete that line, stop caring about gtk2
2) change that line to force an old gtk2 build.

I think, but with no strong opinion, that we might want to keep the gtk2 build to see if something breaks, and maybe ignore failures in case we don't want it anymore, or drop completely the gtk2 code, but as long as maintenance is zero effort...
In the meanwhile, in the next two years, Debian will probably drop gtk2 from sid, and maybe start to support gtk4, that might be a good time to remove all the old code.

What do you think?

Reply:
Ah OK. Now that makes perfectly sense.